### ISSUE TITLE
Missing API endpoint to stop exiting deployment

### ISSUE THREAD

**Issue by mjyeaney** [mjyeaney/aci-deployment-api#10]
Need to add an endpoint to enable clients to stop running deployments. This may look like:

```
PUT /api/deployments/{deployment-name}/status

{status: stopped}
```

However...this may be a little too RPC-ish. Thoughts?

**Comment by mjyeaney** [comment]
Initial pass at this ended up implementing a `DELETE` method...seems like we should have both?

**Comment by mjyeaney** [comment]
New API signature is as follows (based off of the Azure VM REST api):

`POST /api/deployments/{deployment-name}/stopped`

Fixed in PR #16
